fastlane_bot/config/network.py

```diff
@@ -12,7 +12,7 @@
 
 from . import selectors as S
 from .base import ConfigBase
-from .constants import CARBON_V1_FORK1_NAME, CARBON_V1_NAME
+from .constants import CARBON_V1_NAME
```

Comment:
Not clear why this is needed here (no other change in this PR seems to be using it)